stats.rb

```diff
@@ -80,98 +61,99 @@ def tablecreation(title,pr_array)
 total_merged_pulls = 0
 total_mentioned_pulls = 0
 
-  puts "repo, last comment, needs rebase, fails test, needs squash, no comments, total open, has mention, no activty 40 days"
-repos.each do |repo|
-  pr_information_cache = util.fetch_async("#{options[:namespace]}/#{repo}", search_with={:state=>'open', :sort=>'updated'}, filter=[:statuses, :pull_request_commits, :issue_comments, :pull_request])
+puts 'repo, last comment, needs rebase, fails test, needs squash, no comments, total open, has mention, no activty 40 days'
+parsed.each do |m|
+  # Disbled because default value on filter causes github api issues
+  # rubocop:disable Lint/UselessAssignment
+  pr_information_cache = util.fetch_async("#{m['github_namespace']}/#{m['repo_name']}", search_with = { state: 'open', sort: 'updated' }, filter = %i[statuses pull_request_commits issue_comments pull_request])
+
+  closed_pr_information_cache = util.fetch_async("#{m['github_namespace']}/#{m['repo_name']}", search_with = { state: 'closed', sort: 'updated' }, filter = [])
+  # rubocop:enable Lint/UselessAssignment
```

Comment:
rubocop is right here. The assignments to `search_with` and `filter` are useless: the value is not used below. If the intention of this was to use named arguments, please use the correct ruby syntax for this.